### ISSUE TITLE
Merging index fails if DateTime field is included in schema

### ISSUE THREAD

**Issue by mosuka** [quickwit-oss/tantivy#735]
**Describe the bug**
DateTime can be configured as fast field, yet it is possible to configure them as fast fields.

**Which version of tantivy are you using?**
0.11.2

The log is as follows:
```
thread 'merge_thread3' panicked at 'Failed to find a reader for single fast field. This is a tantivy bug and it should never happen.', src/libcore/option.rs:1190:5
```

**Comment by fulmicoton** [comment]
Closed in #736 

Released in 0.11.3

**Comment by mosuka** [comment]
Thanks!
